.github/workflows/test.yml

```diff
@@ -88,53 +88,6 @@ jobs:
           name: "${{ github.sha }}-${{ matrix.part }}-coverage"
           path: ./${{ matrix.part }}profile.out
 
-  upload-coverage-report:
```

Comment:
deleted this cause it keeps breaking, and while it's a nice to have, I wouldn't define it as a must have